src/pal/src/misc/miscpalapi.cpp

```diff
@@ -454,3 +454,159 @@ EnsureOpenSslInitialized()
     return dwRet;
 }
 
+/*++
```

Comment:
@janvorli we want -force_load for OS X or -Wl,--whole-archive for linux to solve that part.